book/website/_toc.yml

```diff
@@ -241,7 +241,8 @@ parts:
       sections:
       - title: Planning for Project Design
         file: project-design/pd-overview/pd-overview-planning
-      - title: Communication and Collaboration
+      - title: 
+      cation and Collaboration
```

Comment:
Oops, looks like something broke here. :) The `Communication and Collaboration` title got moved to a new line.

Reply:
Can I just correct it in my feature branch?

Reply:
Yup, exactly. It should reset the change.